mungegithub/mungers/release-note-label_test.go

```diff
@@ -249,6 +249,11 @@ func TestGetReleaseNote(t *testing.T) {
 			expectedReleaseNoteVariable: releaseNoteNone,
 		},
 		{
+			body:                        "**Release note**:\n\n<other unimportant information> ```NONE```",
```

Comment:
isn't it weird that we skip "<other unimportant information>". What if it was important? (like being something completely different?). The .* becomes VERY lenient now that you have the `(?s)` modifier.

Should we use:
`Release Note:\s*BQ BQ BQ`.....